wiki/Manual.md

```diff
@@ -5520,6 +5520,91 @@ Returns
 }
 ```
 
+
+## Proxy
+
+In some specific cases you may want a proxy, when:
+- Exchange is not available in your location
+- Your IP is forbidden by exchange
+- You experience random restriction by exchange, like [DDoS protection by Cloudflare](https://docs.ccxt.com/#/?id=ddos-protection-by-cloudflare-incapsula)
+
+However, beware that each added intermediary might add some latency to requests.
+
+### Supported proxy types
+CCXT supports the following proxy types:
+
+#### proxyUrl
+
+With this property you can prepend any url to requests, like:
+```
+ex.proxyUrl = 'http://127.0.0.1:8080/';
```

Comment:
@ttodua can we have examples using both "formats" ? (setting the prop after instantiating and upon instantiating), most users do that using the latter.

For instance:
```
ex.proxyUrl = 'http://127.0.0.1:8080/';

// or 

exchange = ccxt.binance({'proxyUrl': 'newValue'})
```

Reply:
done